yb_migrate/src/migration/common.go

```diff
@@ -332,21 +328,16 @@ func CreateMigrationProjectIfNotExists(source *srcdb.Source, exportDir string) {
 	// log.Debugf("Creating a project directory...")
 	//Assuming export directory as a project directory
 	projectDirPath := exportDir
-	if source.GenerateReportMode {
-		projectSubdirs = []string{"temp", "temp/schema", "reports"}
-	}
 
 	for _, subdir := range projectSubdirs {
 		err := exec.Command("mkdir", "-p", projectDirPath+"/"+subdir).Run()
 		utils.CheckError(err, "", "couldn't create sub-directories under "+projectDirPath, true)
 	}
 
 	// Put info to metainfo/schema about the source db
-	if !source.GenerateReportMode {
-		sourceInfoFile := projectDirPath + "/metainfo/schema/" + "source-db-" + source.DBType
-		cmdOutput, err := exec.Command("touch", sourceInfoFile).CombinedOutput()
-		utils.CheckError(err, "", string(cmdOutput), true)
-	}
+	sourceInfoFile := projectDirPath + "/metainfo/schema/" + "source-db-" + source.DBType
```

Comment:
I think we can move this flag to `metainfo/flags` as part of another PR maybe, because this is only thing we are putting in `metainfo/schema` folder as of now.

Reply:
Yes. This can be taken in a separate PR.